SBR.md

```diff
@@ -528,10 +528,12 @@ Pseudonym Certificates are not anonymous. CAs and Enterprise RAs SHALL treat Ind
 
 ### 3.1.4 Rules for interpreting various name forms
 
-In cases where names use diacritics or other characters that are not supported by Relying Party applications, the CA SHOULD define substitution rules in its CP and/or CPS.  For example, regardless of capitalization:
+In cases where a Subject information field (as listed in [Section 7.1.4.2](#7142-subject-information---subscriber-certificates)) contains non-ASCII characters, the CA MAY represent them with ASCII characters. 
 
-*  Accent characters MAY be represented by their ASCII equivalent. For example é, à, í, ñ, or ç MAY be represented by e, a, i, n, or c.
-*  Umlaut-accented characters such as ä, ö, ü MAY be represented by either ae, oe, ue or a, o, u.
+   | Generation | Method to represent non-ASCII characters | 
+   |------|-----------------------|
+   | Strict | The CA MUST use the IDNA2008 |
```

Comment:
[IDNA](https://datatracker.ietf.org/doc/html/rfc5890) is intended for Domain Names and not appropriate for other usages.

Reply:
well, IDNA is a framework of various protocols https://datatracker.ietf.org/doc/html/rfc5890#section-1.1 . We could use [punnycode](https://datatracker.ietf.org/doc/html/rfc3490) (which is part of IDNA) to encode non-ASCII-characters.

Reply:
Punycode is not intended to be readable for humans and its original purpose is to be used in the domain name system, see RFC 3492:

> Punycode is a simple and efficient transfer encoding syntax designed
>   for use with Internationalized Domain Names in Applications (IDNA).
 >  It uniquely and reversibly transforms a Unicode string into an ASCII
 >  string.  ASCII characters in the Unicode string are represented
 >  literally, and non-ASCII characters are represented by ASCII
 >  characters that are allowed in host name labels (letters, digits, and
 >  hyphens).

I suppose the goal is to make the content **understandable for relying parties** (humans). The original concern seems to be that not all relying party software can display these non-ASCII characters, Punycode is not intended to display characters in ASCII but to transport a Unicode string within an ASCII only protocol.

Reply:
> Punycode is not intended to be readable for humans and its original purpose is to be used in the domain name system, see RFC 3492:
> 
> > Punycode is a simple and efficient transfer encoding syntax designed
> > for use with Internationalized Domain Names in Applications (IDNA).
> > It uniquely and reversibly transforms a Unicode string into an ASCII
> > string.  ASCII characters in the Unicode string are represented
> > literally, and non-ASCII characters are represented by ASCII
> > characters that are allowed in host name labels (letters, digits, and
> > hyphens).
> 
> I suppose the goal is to make the content **understandable for relying parties** (humans). The original concern seems to be that not all relying party software can display these non-ASCII characters, Punycode is not intended to display characters in ASCII but to transport a Unicode string within an ASCII only protocol.

The current statement in the SBRs allows to convert certain non-ASCII (diacritic) characters to ASCII characters. I am trying to say that focusing on 'diacritic' characters is only addressing a sub-part of the overall problem: how to translate non-ASCII-characters into ASCII characters. It does not make any sense to have regulation for 'diacritic' characters but not for latin characters which are neither 'diacritic' nor ASCII (for example `ß`) or for non-latin characters. If we want to allow the translation from 'diacritic' characters to ASCII characters, we need to allow this for all kinds of non-ASCII characters. Allowing it only for diacritic characters is arbitrary and anglo-centric.

Reply:
>  It does not make any sense to have regulation for 'diacritic' characters but not for latin characters which are neither 'diacritic' nor ASCII (for example ß) or for non-latin characters.

This is why the current language says "or other characters": it encompasses all other "non-ASCII" cases.

Reply:
Good point. It might make sense to remove the reference to "diacritic characters" in total and only write about "characters not supported by".

Reply:
Maybe we could reach out to the vendors of the `Relying Party applications` if they are interested in adding Punycode support into their applications.

Reply:
What would that solve? Better if they would just support UTF-8.